includes/formFactory/ManageWikiFormFactoryBuilder.php

```diff
@@ -627,6 +627,18 @@ private static function buildDescriptorNamespaces(
 
 			foreach( (array)$config->get( 'ManageWikiNamespacesAdditional' ) as $key => $a ) {
 				if ( ( $a['main'] && $name == 'namespace' || $a['talk'] && $name == 'namespacetalk' ) && ( !in_array( $id, (array)$a['blacklisted'] ) ) ) {
+					if ( is_array( $a['overridedefault'] ) ) {
+						if ( in_array( $id, array_keys( $a['overridedefault'] ) ) ) {
+							foreach ( $a['overridedefault'] as $nID => $value ) {
+								if( $id === $nID ) {
```

Comment:
Foreach isn't necessary.

Reply:
From John:

<JohnLewis> as to find the relevant value in an array of $var =  [ $id = > $val ], it would be easier and cheaper to do a check on $var[$myID], then it is to foreach over the array to check each $id if it is equal to $myID

Reply:
Fixed